utils/build_pydusa_numpy.sh

```diff
@@ -17,12 +17,17 @@ set -xe
 
 RECIPES_DIR=$(cd $(dirname $0)/../recipes && pwd -P)
 
-if [ ${#args} -lt 1 ];then
+number_of_words=$(wc -w <<< "${args[@]}")
+if [ ${number_of_words} -le 0 ];then
```

Comment:
If we are going to add more arguments/options, I would suggest replacing if/else with case statements and adding a `print_usage` function like it's done here, https://github.com/cryoem/build-scripts/blob/master/cronjob.sh.